shopfloor_mobile_base/static/wms/src/main.js

```diff
@@ -231,9 +235,20 @@ new Vue({
                     "SERVICE-CTX-PROFILE-ID": this.profile.id,
                 },
             });
-            return odoo.call("menu").then(function (result) {
-                self.appmenu = result.data;
-            });
+            if (!this.loadingMenu) {
```

Comment:
As discussed together: pls use a specific event to trigger this only when needed.